test/Microsoft.OpenApi.Readers.Tests/NodeTests.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
```

Comment:
nit copyright

Reply:
Dammit.  I need to setup that editorConfig file and see if it can catch stuff like this, because I always forget.

---
In reply to: [194831175](https://github.com/Microsoft/OpenAPI.NET/pull/276#discussion_r194831175) [](ancestors = 194831175)